emcache/protocol.py

```diff
@@ -194,6 +195,21 @@ async def _extract_one_line_data(self, data: bytes):
         finally:
             self._parser = None
 
+    async def _extract_pipeline_data(self, data: bytes):
+        future = self._loop.create_future()
+        parser = cyemcache.AsciiPipeLineParser(future)
+        self._parser = parser
+        self._transport.write(data)
+        self._transport.write(b"mn\r\n")
+        try:
```

Comment:
To be honest, I didn’t understand why we didn’t catch errors, here I tried to be more selective about errors